javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,167 @@
+// Flight booking full name function =============================================================================
+
+const getFullName = (firstName, lastName) => `${firstName} ${lastName}`;
+
+const fullName1 = getFullName('Rostyslav', 'Zhyvitskyi');
+const fullName2 = getFullName('Benjamin', 'Hughes');
+
+console.log(fullName1, fullName2);
+
+// Formal full name
+
+const getFullNameFormal = (firstName, lastName, sex, useFormalName) => {
+  if (firstName === '' || lastName === '') {
+    throw Error('Please make sure you have entered your first and last name');
+  }
+
+  if (useFormalName) {
+    return sex === 'male'
+      ? `Mr. ${firstName} ${lastName}`
+      : `Mrs. ${firstName} ${lastName}`;
+  } else {
+    return `${firstName} ${lastName}`;
+  }
+};
+
+// Event application ===============================================================================================
+
+const weekDays = [
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+  'Sunday',
+];
+
+const getEventWeekday = (remainingDays) => {
```

Comment:
The task can be nicely solved with the switch https://www.w3schools.com/js/js_switch.asp
Similar to what you did in the last exercise.